index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="bar">
+                <p>9:41</p>
+                <img class="mobile_network" src="images/header_img/tower.png" alt="network_img">
+                <img class="mobile_wifi" src="images/header_img/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header_img/Battery.png" alt="battery">
+            </div>
```

Comment:
- [ ] Please note that this bar is not  part of the design and it shouldn't show up in your website , kindly remove it it only added on Figma to show you that the design is on mobile

![image](https://user-images.githubusercontent.com/89282221/192019861-188ff0df-97a3-44a7-b5da-42d4eeff236c.png)